salt/modules/acme.py

```diff
@@ -114,6 +116,8 @@ def cert(name,
     :param aliases: subjectAltNames (Additional DNS names on certificate)
     :param email: e-mail address for interaction with ACME provider
     :param webroot: True or a full path to use to use webroot. Otherwise use standalone mode
+    :param dns_plugin: Name of a DNS plugin to use
+    :param dns_cloudflare_credentials: Path to CloudFlare credentials file if the CloudFlare DNS Plugin is in use
```

Comment:
I'm not a huge fan of adding an argument that's specific to a single provider. Can we make this something like `dns_plugin_credentials`?